fenix/app/src/main/java/org/mozilla/fenix/addons/InstalledAddonDetailsFragment.kt

```diff
@@ -118,10 +118,21 @@ class InstalledAddonDetailsFragment : Fragment() {
         bindRemoveButton()
     }
 
+    @VisibleForTesting
+    internal fun provideEnableSwitch() = binding.enableSwitch
+
+    @VisibleForTesting
+    internal fun providePrivateBrowsingSwitch() = binding.allowInPrivateBrowsingSwitch
+
+    @VisibleForTesting
     @SuppressWarnings("LongMethod")
-    private fun bindEnableSwitch() {
-        val switch = binding.enableSwitch
-        val privateBrowsingSwitch = binding.allowInPrivateBrowsingSwitch
+    internal fun bindEnableSwitch() {
+        val switch = provideEnableSwitch()
+        val privateBrowsingSwitch = providePrivateBrowsingSwitch()
+        if (addon.isDisabledAsBlocklisted()) {
+            switch.isEnabled = false
+            return
+        }
         switch.setState(addon.isEnabled())
```

Comment:
Can we add that before the early return? I think it should help fix this UI glitch:

![image](https://github.com/mozilla-mobile/firefox-android/assets/217628/b9d2d121-fc44-49f3-8e8c-f43ca1a88d5c)

Reply:
Sure updating it!